test/built-ins/Temporal/PlainDateTime/prototype/until/roundingmode-ceil-basic.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.until
+description: Checking that ceiling rounding mode rounds correctly
+features: [Temporal]
+---*/
+
+const earlier = new Temporal.PlainDateTime(2019, 1, 8, 8, 22, 36, 123, 456, 789);
+const later = new Temporal.PlainDateTime(2021, 9, 7, 12, 39, 40, 987, 654, 321);
+
+const incrementOneCeil = [
+  ["years", "P3Y", "-P2Y"],
```

Comment:
It's unfortunate, but I'd try to avoid comparing the `toString()` output in this test (also the ones for the other rounding modes) and use TemporalHelpers.assertDuration instead.

e.g. have something like
```js
const incrementOneCeil = [
  ["years", [3], [-2]],
  ["months", [0, 32], [0, -31]],
  // ...
];
...
const [y, m = 0, w = 0, d = 0, h = 0, min = 0, s = 0, ms = 0, µs = 0, ns = 0] = expectedPositive;
TemporalHelpers.assertDuration(
  earlier.until(later, { smallestUnit, roundingMode: "ceil" }),
  y, m, w, d, h, min, s, ms, µs, ns,
  `rounds up to ${smallestUnit} (roundingMode = ceil, positive case)`
);
```

Reply:
No problem (though I think there's something to be said for more human-readable tests, even if we're testing `toString()`, in addition to `until`). Done. I also took care of the other tests. To make some of the tests more readable, I needed to add a helper function that converts `-0` to `0`.